### PR TITLE
Geopoly extension demo app

### PR DESCRIPTION
**Pull request by unatarajan**
# Overview / Task
- [ ] Bug
- [x] Feature

## Description
Example app to demo SQLite's Geopoly extension.

## Feature Validation / Bug Reproduction Steps
Follow the README's TL;DR section to get up-and-running with the app.

## Screenshots & Videos
Screenshot 2024-08-06 at 3 33 47 PM

### CODE REVIEW

**Comment by unatarajan** [comment]
If no feedback, please only approve, don't merge as yet. I will add a README before merging.

File: geopoly-demo/data/geodata.json
Comment (jacobprall) [inline geopoly-demo/data/geodata.json:1]: Let's drop this from the repo - we can link to the blog post if they want to see how you generated the data!

**Review by jacobprall — APPROVED** [review]

**Pull request merged**
